### PR TITLE
[docs] Add dashboards section

### PR DESCRIPTION
**Pull request by VSevagen**
This commit adds the section about dashboards.
Topics such as filtering data, querying data and
navigating through a dashboard are covered. Content
on creating, editing and removing a dashboard are also
covered.

Signed-off-by: sevagenv <sevagenv@gmail.com>

### CODE REVIEW

File: docs/navigate-dashboards/dashboard-access.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (vchrombie) [inline docs/navigate-dashboards/dashboard-access.md:1]: can we rename to `access.md`?

File: docs/navigate-dashboards/data-filter.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
```
Comment (vchrombie) [inline docs/navigate-dashboards/data-filter.md:1]: can we rename to `filter.md`?

File: docs/navigate-dashboards/query-data.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
```
Comment (vchrombie) [inline docs/navigate-dashboards/query-data.md:1]: can we rename to `query.md`?

**Review by vchrombie — COMMENTED** [review]
I think we can simplify the names here too.

`navigating-dashboards` -> `dashboards`

`Navigating Dashboards` -> `Dashboards`

WDYT?

**Comment by VSevagen** [comment]
> I think we can simplify the names here too.

> `navigating-dashboards -> dashboards`

> `Navigating Dashboards -> Dashboards`

> WDYT?

I think navigating-dashboards is clearer than just dashboards since we will be having another section about 'Creating and Editing Dashboards'. I could understand `navigating-dashboards -> dashboards` for the directory name but I can't agree on `Navigating Dashboards -> Dashboards` for the title.

**Comment by vchrombie** [comment]
> I think navigating-dashboards is clearer than just dashboards since we will be having another section about 'Creating and Editing Dashboards'. I could understand navigating-dashboards -> dashboards for the directory name but I can't agree on Navigating Dashboards -> Dashboards for the title.

You sound right, but at the same time the structure doesn't look proper to me. I mean, I feel the current PR and #176 can be merged into a section called `Dashboards` with the below structure.

dashboards
  - access
  - filter
  - query
  - navigation
  -  add
  - new
  - remove
  - edit
  - viz
    - create
    - edit
    - new
    - remove

If you think the structure looks good, you can update the two PRs according to this. dashboards. You need not close the PR, instead update the relevant sections in the PR.

**Comment by VSevagen** [comment]
@vchrombie I've updated this PR. The TOC looks like the following

dashboards

- access
- filter
- query
- navigate
- add
- edit
- remove

Add and new are the same thing. It just about how to create a dashboard.

**Comment by VSevagen** [comment]
> > I think navigating-dashboards is clearer than just dashboards since we will be having another section about 'Creating and Editing Dashboards'. I could understand navigating-dashboards -> dashboards for the directory name but I can't agree on Navigating Dashboards -> Dashboards for the title.
> 
> You sound right, but at the same time the structure doesn't look proper to me. I mean, I feel the current PR and #176 can be merged into a section called `Dashboards` with the below structure.
> 
> dashboards
> 
> * access
> * filter
> * query
> * navigation
> * add
> * new
> * remove
> * edit
> * viz
>   
>   * create
>   * edit
>   * new
>   * remove
> 
> If you think the structure looks good, you can update the two PRs according to this. dashboards. You need not close the PR, instead update the relevant sections in the PR.

Yep. That looks fine. I just made some slight changes in the TOC. The new and add are both about creating a dashboard so I kept add.

**Review by vchrombie — APPROVED** [review]
LGTM!

@VSevagen thanks for your work. Can you rebase the PR so we can proceed for merging?

**Comment by VSevagen** [comment]
@vchrombie I rebased the PR. Sorry for the late response.

**Comment by vchrombie** [comment]
> @vchrombie I rebased the PR. Sorry for the late response.

No issues, thanks for your efforts.

**Pull request merged**